You missed this:
```
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:192:35: error: not a Doxygen trailing comment [-Wdocumentation]
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:193:35: error: not a Doxygen trailing comment [-Wdocumentation]
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:194:35: error: not a Doxygen trailing comment [-Wdocumentation]
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:195:35: error: not a Doxygen trailing comment [-Wdocumentation]
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:196:35: error: not a Doxygen trailing comment [-Wdocumentation]
  /data/riotbuild/riotbase/boards/p-nucleo-wb55/include/periph_conf.h:197:35: error: not a Doxygen trailing comment [-Wdocumentation]
```

--- a/boards/nucleo-l476rg/include/periph_conf.h
+++ b/boards/nucleo-l476rg/include/periph_conf.h
@@ -231,12 +231,12 @@ static const spi_conf_t spi_config[] = {
  * @{
  */
 static const adc_conf_t adc_config[] = {
-    {GPIO_PIN(PORT_A, 0), 0, 5},  /*< ADC12_IN5 */
-    {GPIO_PIN(PORT_A, 1), 0, 6},  /*< ADC12_IN6 */
-    {GPIO_PIN(PORT_A, 4), 1, 9},  /*< ADC12_IN9 */
-    {GPIO_PIN(PORT_B, 0), 1, 15}, /*< ADC12_IN15 */
-    {GPIO_PIN(PORT_C, 1), 2, 2},  /*< ADC123_IN2 */
-    {GPIO_PIN(PORT_C, 0), 2, 1},  /*< ADC123_IN1 */
+    {GPIO_PIN(PORT_A, 0), 0, 5},  /* ADC12_IN5 */
+    {GPIO_PIN(PORT_A, 1), 0, 6},  /* ADC12_IN6 */
+    {GPIO_PIN(PORT_A, 4), 1, 9},  /* ADC12_IN9 */
+    {GPIO_PIN(PORT_B, 0), 1, 15}, /* ADC12_IN15 */
+    {GPIO_PIN(PORT_C, 1), 2, 2},  /* ADC123_IN2 */
+    {GPIO_PIN(PORT_C, 0), 2, 1},  /* ADC123_IN1 */
     {GPIO_UNDEF, 0, 18}, /* VBAT */
 };
 

--- a/boards/p-nucleo-wb55/include/periph_conf.h
+++ b/boards/p-nucleo-wb55/include/periph_conf.h
@@ -189,12 +189,12 @@ static const stm32_usbdev_fs_config_t stm32_usbdev_fs_config[] = {
  * @{
  */
 static const adc_conf_t adc_config[] = {
-    {GPIO_PIN(PORT_C, 0), 0, 1},  /*< ADC1_IN1 */
-    {GPIO_PIN(PORT_C, 1), 0, 2},  /*< ADC1_IN2 */
-    {GPIO_PIN(PORT_A, 1), 0, 6},  /*< ADC1_IN6 */
-    {GPIO_PIN(PORT_A, 0), 0, 5},  /*< ADC1_IN5 */
-    {GPIO_PIN(PORT_C, 3), 0, 4},  /*< ADC1_IN4 */
-    {GPIO_PIN(PORT_C, 2), 0, 3},  /*< ADC1_IN3 */
+    {GPIO_PIN(PORT_C, 0), 0, 1},  /* ADC1_IN1 */
+    {GPIO_PIN(PORT_C, 1), 0, 2},  /* ADC1_IN2 */
+    {GPIO_PIN(PORT_A, 1), 0, 6},  /* ADC1_IN6 */
+    {GPIO_PIN(PORT_A, 0), 0, 5},  /* ADC1_IN5 */
+    {GPIO_PIN(PORT_C, 3), 0, 4},  /* ADC1_IN4 */
+    {GPIO_PIN(PORT_C, 2), 0, 3},  /* ADC1_IN3 */
     {GPIO_UNDEF, 0, 18}, /* VBAT */
 };
 

--- a/boards/stm32l476g-disco/include/periph_conf.h
+++ b/boards/stm32l476g-disco/include/periph_conf.h
@@ -106,11 +106,11 @@ static const uart_conf_t uart_config[] = {
  * @{
  */
 static const adc_conf_t adc_config[] = {
-    {GPIO_PIN(PORT_A, 0), 0, 5},  /*< ADC12_IN5  */
-    {GPIO_PIN(PORT_A, 5), 0, 10}, /*< ADC12_IN10 */
-    {GPIO_PIN(PORT_A, 1), 0, 6},  /*< ADC12_IN6  */
-    {GPIO_PIN(PORT_A, 2), 0, 7},  /*< ADC12_IN7  */
-    {GPIO_PIN(PORT_A, 3), 0, 8},  /*< ADC12_IN8  */
+    {GPIO_PIN(PORT_A, 0), 0, 5},  /* ADC12_IN5  */
+    {GPIO_PIN(PORT_A, 5), 0, 10}, /* ADC12_IN10 */
+    {GPIO_PIN(PORT_A, 1), 0, 6},  /* ADC12_IN6  */
+    {GPIO_PIN(PORT_A, 2), 0, 7},  /* ADC12_IN7  */
+    {GPIO_PIN(PORT_A, 3), 0, 8},  /* ADC12_IN8  */
     {GPIO_UNDEF, 0, 18}, /* VBAT */
 };
 

--- a/cpu/cc2538/periph/rtt.c
+++ b/cpu/cc2538/periph/rtt.c
@@ -118,7 +118,8 @@ void rtt_set_alarm(uint32_t alarm, rtt_cb_t cb, void *arg)
     alarm_cb = cb;
     alarm_arg = arg;
 
-    DEBUG("rtt_set_alarm(%lu), alarm in %lu ticks, overflow in %lu ticks\n",
+    DEBUG("rtt_set_alarm(%" PRIu32 "), alarm in %" PRIu32 " ticks, overflow in "
+          "%" PRIu32 " ticks\n",
           alarm, rtt_alarm - _rtt_get_counter(),
           rtt_offset - _rtt_get_counter());
 
@@ -200,13 +201,13 @@ void isr_sleepmode(void)
     }
 
     if (alarm_cb && (rtt_offset >= rtt_alarm)) {
-        DEBUG("rtt: next alarm in %lu ticks (RTT)\n",
+        DEBUG("rtt: next alarm in %" PRIu32 " ticks (RTT)\n",
               rtt_alarm - _rtt_get_counter());
 
         rtt_next_alarm = RTT_ALARM;
         _set_alarm(rtt_alarm);
     } else if (overflow_cb && (rtt_alarm > rtt_offset)) {
-        DEBUG("rtt: next alarm in %lu ticks (OVERFLOW)\n",
+        DEBUG("rtt: next alarm in %" PRIu32 " ticks (OVERFLOW)\n",
                rtt_offset - _rtt_get_counter());
 
         rtt_next_alarm = RTT_OVERFLOW;

--- a/cpu/cc2538/periph/timer.c
+++ b/cpu/cc2538/periph/timer.c
@@ -185,7 +185,7 @@ static inline cc2538_gptimer_t *dev(tim_t tim)
  */
 int timer_init(tim_t tim, uint32_t freq, timer_cb_t cb, void *arg)
 {
-    DEBUG("%s(%u, %lu, %p, %p)\n", __FUNCTION__, tim, freq, cb, arg);
+    DEBUG("%s(%u, %" PRIu32 ", %p, %p)\n", __FUNCTION__, tim, freq, cb, arg);
 
     if (tim >= TIMER_NUMOF) {
         return -1;
@@ -362,8 +362,8 @@ void timer_start(tim_t tim)
 /**
  * @brief   timer interrupt handler
  *
- * @param[in] num   GPT instance number
- * @param[in] chn   channel number (0=A, 1=B)
+ * @param[in]   tim     timer
+ * @param[in]   channel channel number (0=A, 1=B)
  */
 static void irq_handler(tim_t tim, int channel)
 {

--- a/cpu/gd32v/periph/wdt.c
+++ b/cpu/gd32v/periph/wdt.c
@@ -129,7 +129,7 @@ void wdt_setup_reboot(uint32_t min_time, uint32_t max_time)
     _set_prescaler(pre);
     _set_reload(rel);
 
-    DEBUG("[wdt]: reset time %lu [us]\n", _wdt_time(pre, rel));
+    DEBUG("[wdt]: reset time %" PRIu32 " [us]\n", _wdt_time(pre, rel));
 
     /* Refresh wdt counter */
     wdt_kick();

--- a/cpu/nrf5x_common/periph/uart.c
+++ b/cpu/nrf5x_common/periph/uart.c
@@ -112,6 +112,7 @@ static inline void set_power(uart_t uart, bool value)
 #endif
 }
 
+MAYBE_UNUSED
 static inline bool get_power(uart_t uart)
 {
     UART_TYPE *dev = uart_config[uart].dev;

--- a/cpu/riscv_common/irq_arch.c
+++ b/cpu/riscv_common/irq_arch.c
@@ -15,23 +15,21 @@
  * @}
  */
 
-#include <stdio.h>
 #include <inttypes.h>
+#include <stdio.h>
 
-#include "macros/xtstr.h"
-#include "cpu.h"
+#include "architecture.h"
+#include "clic.h"
 #include "context_frame.h"
+#include "cpu.h"
 #include "irq.h"
 #include "irq_arch.h"
+#include "macros/xtstr.h"
 #include "panic.h"
-#include "sched.h"
 #include "plic.h"
-#include "clic.h"
-#include "architecture.h"
-
-#include "xh3irq.h"
-
+#include "sched.h"
 #include "vendor/riscv_csr.h"
+#include "xh3irq.h"
 
 /* Default state of mstatus register */
 #define MSTATUS_DEFAULT (MSTATUS_MPP | MSTATUS_MPIE)
@@ -122,10 +120,10 @@ __attribute((used)) static void handle_trap(uword_t mcause)
         };
 
         if (trap < ARRAY_SIZE(error_messages)) {
-            printf("Machine Cause Error 0x%lx: %s\n", trap, error_messages[trap]);
+            printf("Machine Cause Error 0x%" PRIxWORD ": %s\n", trap, error_messages[trap]);
         }
         else {
-            printf("Machine Cause Error 0x%lx: Reserved / Custom\n", trap);
+            printf("Machine Cause Error 0x%" PRIxWORD ": Reserved / Custom\n", trap);
         }
     }
 #endif

--- a/cpu/stm32/Makefile.include
+++ b/cpu/stm32/Makefile.include
@@ -94,7 +94,7 @@ ifneq (,$(filter STM32F030x4 STM32MP157Cxx,$(CPU_LINE)))
 else
   STM32CMSIS_INCLUDE_DIR = $(BUILD_DIR)/stm32/cmsis/$(CPU_FAM)/Include
   STM32FAM_INCLUDE_FILE = $(STM32CMSIS_INCLUDE_DIR)/stm32$(CPU_FAM)xx.h
-  INCLUDES += -I$(STM32CMSIS_INCLUDE_DIR)
+  INCLUDES += -isystem$(STM32CMSIS_INCLUDE_DIR)
 endif
 
 # Fetch all CMSIS headers using the package mechanism. This rule is called all

--- a/drivers/kw41zrf/kw41zrf_xcvr.c
+++ b/drivers/kw41zrf/kw41zrf_xcvr.c
@@ -884,8 +884,8 @@ int kw41zrf_xcvr_init(kw41zrf_t *dev)
     /* Load IFR trim values */
     IFR_SW_TRIM_TBL_ENTRY_T sw_trim_tbl[] =
     {
-        {TRIM_STATUS, 0, 0}, /*< Fetch the trim status word if available.*/
-        {TRIM_VERSION, 0, 0} /*< Fetch the trim version number if available.*/
+        {TRIM_STATUS, 0, 0}, /* Fetch the trim status word if available. */
+        {TRIM_VERSION, 0, 0} /* Fetch the trim version number if available. */
     };
     handle_ifr(&sw_trim_tbl[0], ARRAY_SIZE(sw_trim_tbl));
     DEBUG("[kw41zrf] sw_trim_tbl:\n");

--- a/pkg/nmsis_sdk/Makefile.include
+++ b/pkg/nmsis_sdk/Makefile.include
@@ -1,3 +1,3 @@
 PSEUDOMODULES += nmsis_sdk
 
-INCLUDES += -I$(PKGDIRBASE)/nmsis_sdk/NMSIS/Core/Include
+INCLUDES += -isystem$(PKGDIRBASE)/nmsis_sdk/NMSIS/Core/Include

